scripts/network-launch/launch.sh

```diff
@@ -102,6 +137,34 @@ start_node () {
 EOT
 }
 
+start_node_backup () {
```

Comment:
Let's discuss how we can get rid of this method. I see few changes in it compared with the original:
1. Get`localPeerId` - you will fix its getting in another PR
2. You don't need to introduce a new variable `bootNodeBackupHost`. Let's update the existing `bootNode*` variables with the new host and IP when scripts are launching for backup